serverless/admin_guide/serverless-ossm-setup.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-setup"]
+= Integrating Service Mesh with OpenShift Serverless
+:context: serverless-ossm-setup
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Using {ProductShortName} with {ServerlessProductName} enables developers to configure additional networking and routing options.
+
+The {ServerlessOperatorName} provides Kourier as the default ingress for Knative. However, you can use {ProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. The following procedures provide instructions for integrating {ProductShortName} with or without Kourier enabled.
+
+[IMPORTANT]
+====
+{ServerlessProductName} only supports the use of {ProductName} functionality that is explicitly documented in this guide, and does not support other undocumented features.
+====
+
+// without kourier
+[id="serverless-ossm-setup-native"]
+== Integrating {ProductShortName} with {ServerlessProductName} natively
+
+Integrating {ProductShortName} with {ServerlessProductName} natively, without Kourier, allows you to use additional networking and routing options that are not supported by the default Kourier ingress, such as mTLS functionality.
+
+:FeatureName: mTLS for {ServerlessProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+The examples in the following procedures use the domain `example.com`. The example certificate for this domain is used as a certificate authority (CA) that signs the subdomain certificate.
+
+To complete and verify these procedures in your deployment, you need either a certificate signed by a widely trusted public CA, or a CA provided by your organization. Example commands must be adjusted according to your domain, subdomain and CA.
```

Comment:
I don't think you need the comma after CA. Makes it seem that _or a CA provided by your organization_ an explanation of _widely trusted public CA_.

Reply:
Being picky (love that Oxford comma!) 
s/subdomain/subdomain,